esp-wifi/src/lib.rs

```diff
@@ -459,6 +459,9 @@ pub fn initialize(
 /// currently no way to track whether a peripheral has been initialized,
 /// so deinitialization is done based on the activated feature (`wifi`, `ble`
 /// and/or `coex`).
+/// Before deinitializing, chips with NPL bluetooth (esp32c2, esp32c6, esp32h2)
```

Comment:
"Before deinitializing, chips with NPL bluetooth (esp32c2, esp32c6, esp32h2) users must make sure to stop BLE advertising before."